README.md

```diff
@@ -114,6 +114,21 @@ sent to the callback URL:
     "details" : "The file has a virus"
 }
 ```
+##Whitelisting client services
```

Comment:
Missing space between hashes and text, should be:
## Whitelisting client services